src/app/routes/keycloakSignInRoutes.js

```diff
@@ -0,0 +1,84 @@
+const bodyParser = require('body-parser');
+const envHelper = require('../helpers/environmentVariablesHelper.js');
+const request = require('request-promise');
+var qs = require('querystring');
+var DOMParser = require('xmldom').DOMParser;
+const parser = new DOMParser();
+
+module.exports = (app) => {
+  app.post('/keycloak/login', bodyParser.json(), async (req, res) => {
+    const userName = req.body.emailId;
+    const pswd = req.body.password;
+    let redirect_uri = "";
+    let queryArr = [];
+    (req.body.loginConfig.params).forEach(item => {
+      if (item['key'] == 'redirect_uri') {
+        redirect_uri = item['value'];
+      }
+      queryArr.push((Object.values(item)))
+    });
+    const opt = {
+      method: 'POST',
+      url: req.body.loginConfig.host+req.body.loginConfig.path,
+      headers: {
+        "Content-type": "application/x-www-form-urlencoded",
```

Comment:
do we need this if yes can it be a variable

Reply:
Is this mention for url property.